ddsrouter_core/include/ddsrouter_core/types/endpoint/Endpoint.hpp

```diff
@@ -29,12 +29,21 @@ namespace ddsrouter {
 namespace core {
 namespace types {
 
+using EndpointKindType = unsigned int;
+
 //! Possible kinds of the endpoint
-enum EndpointKind
+enum class EndpointKind : EndpointKindType
+{
+    invalid = 0,
+    writer = 1,
+    reader = 2,
+};
+
+constexpr std::array<const char*, 3> EndpointKindStrings =
```

Comment:
Use project common format for variable names.
Nevertheless, we could open a discussion about changing the names format of const variables.
```suggestion
constexpr std::array<const char*, 3> ENDPOINT_KIND_STRING =
```